doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -249,83 +253,93 @@ Induction
+   :n:`@induction_principle`
+     :n:`one_term` specifies which :term:`induction principle` to use and, optionally,
+     what bindings
      to apply to the induction principle.  The number of :n:`@induction_clause`\s
      must be the same as the number of parameters of the induction principle.
 
-   Other parameters behave as they do in :tacn:`destruct`.
+     If unspecified, the tactic applies the appropriate :term:`induction principle`
+     that was automatically generated when the inductive type was declared based
+     on the type of the goal.
 
-.. example::
+   .. exn:: Not an inductive product.
+      :undocumented:
 
-   .. coqtop:: reset all
+   .. exn:: Unable to find an instance for the variables @ident … @ident.
 
-      Lemma induction_test : forall n:nat, n = n -> n <= n.
-      intros n H.
-      induction n.
-      exact (le_n 0).
+      Use the :n:`with @bindings` clause or the :tacn:`einduction` tactic instead.
 
-.. example:: :tacn:`induction` with :n:`@occurrences`
+   .. example::
 
-   .. coqtop:: reset all
+      .. coqtop:: reset all
 
-      Lemma comm x y : x + y = y + x.
-      induction y in x |-   *.
-      Show 2.
+         Lemma induction_test : forall n:nat, n = n -> n <= n.
+         intros n H.
+         induction n.
+         exact (le_n 0).
 
-   .. exn:: Not an inductive product.
-      :undocumented:
+   .. example:: :tacn:`induction` with :n:`@occurrences`
 
-   .. exn:: Unable to find an instance for the variables @ident … @ident.
+      .. coqtop:: reset all
 
-      Use the variant :tacn:`elim` `… with …` instead.
+         Lemma induction_test2 : forall n:nat, n = n -> n <= n.
+         intros.
+         induction n in H |-.
+         Show 2.
 
-.. tacn:: einduction {+, @induction_clause } {? @induction_principle }
+   .. tacn:: einduction {+, @induction_clause } {? @induction_principle }
 
-   Behaves like :tacn:`induction` except that it does not fail if
-   some dependent premise of the type of :n:`@one_term` is not inferrable. Instead,
-   the unresolved premises are posed as existential variables to be inferred
-   later, in the same way as :tacn:`eapply` does.
+      Behaves like :tacn:`induction` except that it does not fail if
+      some dependent premise of the type of :n:`@one_term` is not inferrable. Instead,
+      the unresolved premises are posed as existential variables to be inferred
+      later, in the same way as :tacn:`eapply` does.
 
 .. tacn:: elim @one_term_with_bindings {? using @one_term {? with @bindings } }
 
-   An older, more basic induction tactic.  We recommend using :tacn:`induction`
-   instead where possible.  :n:`@one_term` must have an inductive
+   An older, more basic induction tactic.  Unlike :tacn:`induction`, ``elim`` only
+   modifies the goal; it does not modify the :term:`local context`.  We recommend
+   using :tacn:`induction` instead where possible.  :n:`@one_term` must have an inductive
```

Comment:
Since everything else also applies to `induction`, the main description of `elim` could really stop after this second sentence.

Reply:
I trimmed only this paragraph and kept "We recommend using induction...".  Did you want me to drop more here (e.g. up to `tacn:: eelim`)?

Reply:
You can also remove the paragraph:

>More generally, elim [...] and otherwise fails.

Since it's a repetition of what is (now) said for `induction`.